### PR TITLE
Enroll 2FA from settings panel

### PR DESCRIPTION
**Pull request by qrtp**
From the settings panel, allow the user to enable / disable the 2FA feature. Currently gated by a feature flag, until we are ready to roll this out as part of the sign-in experience.

## Demo
https://github.com/user-attachments/assets/8b56d51d-e5fc-4b9f-8018-310c191972e4

### CODE REVIEW

File: src/pages/Wallet/TwoFactorModal.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    }
+    const loadStatus = async () => {
+      // retrieve status
+      const enabled = await getTwoFactorStatus(accessToken);
+      updateStatus(enabled);
+
+      // if not enabled, retrieve QR code data
+      if (!enabled) {
+        const challenge = await getTwoFactorChallenge(accessToken);
+        const encodedChallenge = `otpauth://totp/${emailAddress}?secret=${challenge}`;
```
Comment (tirish) [inline src/pages/Wallet/TwoFactorModal.tsx:62]: We should have "Unstoppable Light Wallet" or something in here too

https://github.com/google/google-authenticator/wiki/Key-Uri-Format

Comment (qrtp) [reply src/pages/Wallet/TwoFactorModal.tsx:62]: nice, yeah looks like we could add `&issuer=Unstoppable%20Lite%20Wallet` to the end of the encoded data.

Comment (qrtp) [reply src/pages/Wallet/TwoFactorModal.tsx:62]: Followup, attempted to add `issuer` as both a query parameter and an encoded prefix to the user (e.g. `name:user`). Both methods fail to register in the OTP apps as a code when scanning the QR code. So I think we should leave this out of the encoding.

Comment (tirish) [reply src/pages/Wallet/TwoFactorModal.tsx:62]: you might need to `encodeURIComponent(...)` each of the things you are putting into this uri

Comment (qrtp) [reply src/pages/Wallet/TwoFactorModal.tsx]: good idea, updated 👍

**Pull request merged**